core/src/jmh/java/io/grpc/internal/SerializingExecutorBenchmark.java

```diff
@@ -41,7 +41,7 @@
 public class SerializingExecutorBenchmark {
 
   private ExecutorService executorService = Executors.newSingleThreadExecutor();
-  private Executor executor = new SerializingExecutor(executorService);
+  private Executor executor = SerializingExecutors.wrap(executorService);
```

Comment:
This type can be SerializingExecutor to be more precise, since it's the purpose of the benchmark